nixos/modules/services/misc/mpdscribble.nix

```diff
@@ -0,0 +1,288 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mpdscribble;
+
+  mkSection = cfg: optname: secname:
+    if cfg."${optname}" then
+      ''
+        [${secname}]
+        url      = ${cfg."${optname}".url}
+        username = ${cfg."${optname}".username}
+        password = ${cfg."${optname}".password}
+        journal  = ${cfg."${optname}".journal}
+      '' else "";
+
+  cfgFile = pkgs.writeText "mpdscribble.conf" ''
+## mpdscribble - an audioscrobbler for the Music Player Daemon.
+## http://mpd.wikia.com/wiki/Client:mpdscribble
+
+# HTTP proxy URL.
+${if isNull cfg.proxy then "" else "proxy = ${cfg.proxy}"}
```

Comment:
Is it deliberate that you do not use `networking.proxy`?

Reply:
What do you mean?

Also: This PR is stalled too long, but I can't get it finished... am starting `mpdscribble` after each boot by hand (which is ugly and sometimes I forget it)... so if someone wants to take this and finish it: feel free!

Reply:
I mean, you configure a proxy, but there is a system-wide proxy configuration variable.

I am not using `mpd` at all, I just hoped to merge a random sample of long-standing PRs.